src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetRandomCommand.cs

```diff
@@ -361,113 +361,108 @@ private Int64 GetRandomInt64(Int64 min, Int64 max)
             do
             {
                 // Randomly fill the buffer
-                this.Generator.NextBytes(buffer);
+                Generator.NextBytes(buffer);
                 randomUint64 = BitConverter.ToUInt64(buffer, 0);
-                // Get the last 'bitsToRepresentDiff' number of randon bits
+                // Get the last 'bitsToRepresentDiff' number of random bits
```

Comment:
Please insert new line.
```suggestion

                // Get the last 'bitsToRepresentDiff' number of random bits
```